workspaces/announcements/plugins/announcements-backend/src/router.ts

```diff
@@ -218,9 +228,13 @@ export async function createRouter(
           metadata: { action: EVENTS_ACTION_CREATE_ANNOUNCEMENT },
         });
 
-        await signalAnnouncement(announcement, signals);
+        if (isSignalsEnabled) {
+          await signalAnnouncement(announcement, signals);
+        }
+        if (isAnnouncementNotificationsEnabled) {
+          sendAnnouncementNotification(announcement, notifications);
+        }
```

Comment:
Hey @drodil - do you have time to take a look at this? I'm worried we are going to duplicate signals. 

Should we do a conditional if/elseif here? Meaning if they have announcements enabled, we should only call `sendAnnouncementNotification` and not `signalAnnouncement`. If announcements are disabled, then handle signals like we do currently.

Reply:
Hi. It depends. If both are enabled, the UI will show both the announcement and the notification in real time (banner on top of the screen for announcement and pop-up in the lower right corner for notification). 

At the moment, there's no possibility to skip the signal sending in the notifications plugin which IMO would be the right way so that people who are not in the portal at the time of the sending would get a notification about the announcement and see it once they log in. 

Without the possibility to skip the signal sending in notifications, I would send both in this case (of course based on config), add support for this feature in the backstage, and later use it here for notifications. WDYT?

Reply:
Leaning towards a simple config option to enable/disable sending a notification being sufficient. If you enable notifications on the announcements plugins, it can be assumed you will get both. 

I appreciate you taking the time to review.